fastify.js

```diff
@@ -245,6 +262,7 @@ function fastify (options) {
     // custom error handling
     setNotFoundHandler: setNotFoundHandler,
     setErrorHandler: setErrorHandler,
+    errorHandler: defaultErrorHandler,
```

Comment:
this should be a getter instead, returning `kErrorHandler`. You should also replace all the usage of `_errorHandler` with `kErrorHandler`.